airbyte-integrations/connectors/source-leadfeeder/main.py

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+
+import sys
+
+from airbyte_cdk.entrypoint import launch
```

Comment:
It's better to import `run` and just run it. But I understand this is templated, alright to keep.